caom2pipe/tests/test_run_composable.py

```diff
@@ -1288,8 +1288,9 @@ def _post(self):
         assert self._reporter._summary._skipped_sum == 0, 'skipped'
         assert self._reporter._summary._timeouts_sum == 0, 'timeout'
 
+    @patch('caom2pipe.data_source_composable.CadcTapClient')
```

Comment:
I don't see the mock for the tap client being defined in any of these functions.

Reply:
Good catch - it's not necessary.